arguments.py

```diff
@@ -1,31 +1,52 @@
 import os
-from logger import logger, configure_logger
+from logger import configure_logger
```

Comment:
It would seem nicer to separate the commit in an automated part vs. manual modification.

One might assume, that the tool behaves correctly (or at least, it does what it does). So reviewing the manual changes would be especially interesting. But it's hard to see which changes were done by hand.

Or were the unnecessary imports removed by a tool? How?

In any case, the command line you used to make this commit would be nice too. I presume it's just `black .`?

Reply:
Yes. I'll put that in a separate PR